packages/amplify-category-function/provider-utils/awscloudformation/utils/cron.js

```diff
@@ -0,0 +1,267 @@
+const path = require('path');
```

Comment:
I know you made this file and other .js because that's what everything in the functions plugin is right now. But my refactor is going to change all of this to .ts so it would be great if this could also be .ts :)